com.termius.termius.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2019 Termius Corporation.> -->
+<component>
+ <id>com.termius.termius</id>
+ <metadata_license>CC0-1.0</metadata_license>
```

Comment:
You need a 
`<launchable type="desktop-id">com.termius.termius.desktop</launchable>`
here

Reply:
Ok have try a fix